fix - make editor scale to user's code length up to 1000 lines

Z4CHx opened the pull request:
edit editor css and editor settings to be relatively positioned, which
keeps the layout in tact while letting the box expand to up to 1000
lines.

Z4CHx committed:
make editor relative and add code line limit

edit editor css and editor settings to be relatively positioned, which
keeps the layout in tact while letting the box expand to up to 1000
lines.

eliotc1986 commented:
<img width="1147" alt="screen shot 2017-04-13 at 2 58 35 pm" src="https://cloud.githubusercontent.com/assets/6524512/24992021/de9ab866-2059-11e7-895b-45791ce77141.png">

It looks like you need to remove the min-height on the `#editor-wrapper` or else short posts get this sort of look

Code can be found here:

https://github.com/eliotc1986/algo-app/blob/668279fe9b4be8b298ab4185de1e4b18d781ba54/app/assets/stylesheets/views/_code_editor.scss#L4

Z4CHx committed:
fix editor height with short bodies of code

remove min-height from editor-wrapper to maintain layout when the
user’s code is short.

shin10kudev commented on app/assets/stylesheets/views/_code_editor.scss:
```diff
@@ -1,7 +1,7 @@
 // Code editor styles
 
 #editor-wrapper {
```
Please remove the entire selector

shin10kudev replied:
It's good practice to remove empty id and class selectors

Z4CHx committed:
remove empty editor style from prior fix

eliotc1986 commented:
LGTM 👍
